pages/common/qutebrowser.md

```diff
@@ -0,0 +1,24 @@
+# qutebrowser
+
+> A keyboard-driven, vim-like browser based on PyQt5.
+> More information: <https://qutebrowser.org/>.
+
+- Open qutebrowser with a specified (non-default) storage directory:
+
+`qutebrowser --basedir {{base_directory}}`
+
+- Open a qutebrowser instance with temporary settings:
+
+`qutebrowser --set {{option}} {{value}}`
```

Comment:
Could you add an example of listing the available options?  Without that, this example might not be very useful.